<!DOCTYPE HTML>
<!--
	Stellar by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Abolfazl Doostparast</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
		
		
	<meta name="keywords" content="Abolfazl,Doostparast,Torshizi,Abolfazl Doostparast">	

<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-135269487-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-135269487-1');
</script>
<script src="https://code.iconify.design/1/1.0.2/iconify.min.js"></script>
		
		  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/v4-shims.css">


		
		
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
					<header id="header" class="alt">
						<!--<span class="logo"><img src="images/pic011.jpg" alt="" /></span> -->
						<h1>Abolfazl Doostparast Torshizi</h1>
						
						<!-- built by <a href="https://twitter.com/ajlkn">@ajlkn</a> for <a href="https://html5up.net">HTML5 UP</a>.</p> -->
					</header>

				<!-- Nav -->
					<nav id="nav">
						<ul>
							<li><a href="#intro" class="active">Introduction</a></li>
							<li><a href="#pub" class="active">Publications</a></li>
							<li><a href="#news">News & Media</a></li>
							<li><a href="#fund">Funding & Awards</a></li>
							<li><a href="#servis">Service</a></li>
							<li><a href="#ment">Mentoring Experience</a></li>
							<li><a href="#first">Social Footprints</a></li>
							<li><a href="#contact">Contact</a></li>
							
						</ul>
					</nav>

				<!-- Main -->
					<div id="main">

						<!-- Introduction -->
							<section id="intro" class="main">
								<div class="spotlight">
									<div class="content">
									<!--	<header class="major">
											<h2>Abolfazl Doostparast Torshizi</h2>                                                             
										</header> --> 
										<p>My name is <strong>Abolfazl</strong>. I am a senior principal scientist II at Johnson and Johnson Innovative Medicine. Previously, I was a senior research scientist at  <a href="https://www.med.upenn.edu/">Perelman School of Medicine</a> at the <a href="https://home.www.upenn.edu/">University of Pennsylvania</a>. I was
									also a member of <a href="https://ccmt.research.chop.edu/">Raymond G. Perelman Center for Cellular and Molecular Therapeutics</a> at the <a href="https://www.chop.edu/">Children's Hospital of Philadelphia</a>.</p> <p>Prior to moving to Philadelphia, we were a part of the <a href="https://www.dbmi.columbia.edu/">Department of Biomedical Informatics</a> and the <a href="http://www.igm.columbia.edu/">Institute for Genomic Medicine</a> at Columbia
 University. I did my postdoc in Computer Science and Systems Biology at the University of California, Santa Barbara and Harvard University. I earned my PhD in Industrial and Systems Engineering (minor in System biology) from <a href="https://aut.ac.ir/en">Amirkabir University of Technology</a> (former Tehran Polytechnic), Iran, in 2015.
 I am broadly interested in Bioinformatics, Computational Systems Biology, Machine Learning, Data Mining and Modeling, and Single-Cell Genomics in the context of complex disorders.</p>
										<!-- <ul class="actions" >
											<li><a href="generic.html" class="button">Learn More</a></li>
										</ul> -->
									</div>
									<span class="image"><img src="images/pic0111.jpg" alt="" /></span>
								</div>
								
		                    <section id="pub" class="main ">
                              <header class="major spacial">
									<h2>Publications</h2>
								<p>Here I have shortlisted my recent publications in chronological order. Full list is available on <a href="https://scholar.google.com/citations?user=4hEOYvcAAAAJ&hl=en">Google Scholar</a>.</p>
								
																		
								
								<!-- width="300" height="400" -->
								</header>
								 <header class="major">

								<p> <div class="col-3"><span class="image left"><img src="images/NatCom.jpg"  alt="" /></span> <em><b> Whole-genome sequencing of 490,640 UK Biobank participants.</b></em>  Whole-genome sequencing provides an unbiased and complete view of the human genome and enables the discovery of genetic variation without the technical limitations of other genotyping technologies. Here we report on whole-genome sequencing of 490,640 UK Biobank participants, building on previous genotyping effort1. This advance deepens our understanding of how genetics associates with disease biology and further enhances the value of this open resource for the study of human biology and health. Coupling this dataset with rich phenotypic data, we surveyed within- and cross-ancestry genomic associations and identified novel genetic and clinical insights. Although most associations with disease traits were primarily observed in individuals of European ancestries, strong or novel signals were also identified in individuals of African and Asian ancestries. With the improved ability to accurately genotype structural variants and exonic variation in both coding and UTR sequences, we strengthened and revealed novel insights relative to whole-exome sequencing2,3 analyses. This dataset, representing a large collection of whole-genome sequencing data that is available to the UK Biobank research community, will enable advances of our understanding of the human genome, facilitate the discovery of diagnostics and therapeutics with higher efficacy and improved safety profile, and enable precision medicine strategies with the potential to improve global health. <a href="https://www.nature.com/articles/s41586-025-09272-9#citeas"><span class="label">(Paper URL)</span></a>.</div></p> 	 	 
								<p> <div class="col-3"><span class="image left"><img src="images/NatCom.jpg"  alt="" /></span> <em><b> Proteogenomic network analysis reveals dysregulated mechanisms and potential mediators in Parkinson’s disease.</b></em>  Parkinson’s disease is highly heterogeneous across disease symptoms, clinical manifestations and progression trajectories, hampering the identification of therapeutic targets. Despite knowledge gleaned from genetics analysis, dysregulated proteome mechanisms stemming from genetic aberrations remain underexplored. In this study, we develop a three-phase system-level proteogenomic analytical framework to characterize disease-associated proteins and dysregulated mechanisms. Proteogenomic analysis identified 577 proteins that enrich for Parkinson’s disease-related pathways, such as cytokine receptor interactions and lysosomal function. Converging lines of evidence identified nine proteins, including LGALS3, CSNK2A1, SMPD3, STX4, APOA2, PAFAH1B3, LDLR, HSPB1, BRK1, with potential roles in disease pathogenesis. This study leverages the largest population-scale proteomics dataset, the UK Biobank Pharma Proteomics Project, to characterize genetically-driven protein disturbances associated with Parkinson’s disease. Taken together, our work contributes to better understanding of genome-proteome dynamics in Parkinson’s disease and sets a paradigm to identify potential indirect mediators connected to GWAS signals for complex neurodegenerative disorders. <a href="https://www.nature.com/articles/s41467-024-50718-x"><span class="label">(Paper URL)</span></a>.</div></p> 	 
							         <p> <div class="col-3"><span class="image left"><img src="images/7982.png"  alt="" /></span> <em><b> Plasma proteomic associations with genetics and health in the UK Biobank.</b></em> The Pharma Proteomics Project is a precompetitive biopharmaceutical consortium characterizing the plasma proteomic profiles of 54,219 UK Biobank participants. Here we provide a detailed summary of this initiative, including technical and biological validations, insights into proteomic disease signatures, and prediction modelling for various demographic and health indicators. We present comprehensive protein quantitative trait locus (pQTL) mapping of 2,923 proteins that identifies 14,287 primary genetic associations, of which 81% are previously undescribed, alongside ancestry-specific pQTL mapping in non-European individuals. The study provides an updated characterization of the genetic architecture of the plasma proteome, contextualized with projected pQTL discovery rates as sample sizes and proteomic assay coverages increase over time. We offer extensive insights into trans pQTLs across multiple biological domains, highlight genetic influences on ligand–receptor interactions and pathway perturbations across a diverse collection of cytokines and complement networks, and illustrate long-range epistatic effects of ABO blood group and FUT2 secretor status on proteins with gastrointestinal tissue-enriched expression. We demonstrate the utility of these data for drug discovery by extending the genetic proxied effects of protein targets, such as PCSK9, on additional endpoints, and disentangle specific genes and proteins perturbed at loci associated with COVID-19 susceptibility. This public–private partnership provides the scientific community with an open-access proteomics resource of considerable breadth and depth to help to elucidate the biological mechanisms underlying proteo-genomic discoveries and accelerate the development of biomarkers, predictive models and therapeutics. <a href="https://www.nature.com/articles/s41586-023-06592-6"><span class="label">(Paper URL)</span></a>.</div></p> 
								 <p> <div class="col-3"><span class="image left"><img src="images/SA.gif"  alt="" /></span> <span style="text-decoration: underline;">A. Doostparast Torshizi</span>, et al.,<em><b> Deconvolution of Transcriptional Networks Identifies TCF4 as a Master Regulator in Schizophrenia.</b></em> Tissue-specific reverse engineering of transcriptional networks has uncovered master regulators (MRs) of cellular networks in various cancers, yet the application of this method to neuropsychiatric disorders is largely unexplored. Here, using RNA-Seq data on postmortem dorsolateral prefrontal cortex (DLPFC) from schizophrenia (SCZ) patients and control subjects, we deconvolved the transcriptional network to identify MRs that mediate expression of a large body of target genes. Together with an independent RNA-Seq data on cultured cells derived from olfactory neuroepithelium, we identified TCF4, a leading SCZ risk locus implicated by genome-wide association studies, as one of the top candidate MRs that may be potentially dysregulated in SCZ. We validated the dysregulated TCF4-related transcriptional network through examining the transcription factor binding footprints inferred from human induced pluripotent stem cell (hiPSC)-derived neuronal ATAC-Seq data <a href="https://advances.sciencemag.org/content/5/9/eaau4139"><span class="label">(Paper URL)</span></a>.</div></p> 
								 <p> <div class="col-3"><span class="image left"><img src="images/Patterns.jpg"  alt="" /></span> <span style="text-decoration: underline;">A. Doostparast Torshizi</span>, et al.,<em><b> Cell-Type-Specific Proteogenomic Signal Diffusion for Integrating Multi-Omics Data Predicts Novel Schizophrenia Risk Genes.</b></em> Accumulation of diverse types of omics data on schizophrenia (SCZ) requires a systems approach to model the interplay between genome, transcriptome, and proteome. We introduce Markov affinity-based proteogenomic signal diffusion (MAPSD), a method to model intra-cellular protein trafficking paradigms and tissue-wise single-cell protein abundances. MAPSD integrates multi-omics data to amplify the signals at SCZ risk loci with small effect sizes, and reveal convergent disease-associated gene modules in the brain. We predicted a set of high-confidence SCZ risk loci followed by characterizing the subcellular localization of proteins encoded by candidate SCZ risk genes, and illustrated that most are enriched in neuronal cells in the cerebral cortex as well as Purkinje cells in the cerebellum. We demonstrated how the identified genes may be involved in neurodevelopment, how they may alter SCZ-related biological pathways, and how they facilitate drug repurposing. MAPSD is applicable in other polygenic diseases and can facilitate our understanding of disease mechanisms. <a href="https://www.cell.com/patterns/fulltext/S2666-3899(20)30119-7"><span class="label">(Paper URL)</span></a>.</div></p> 
								
								  <p> <div class="col-3"><span class="image left"><img src="images/Frontier.jpg"  alt="" /></span><span style="text-decoration: underline;">A. Doostparast Torshizi</span>, I. Ionita-Laza, K. Wang,<em> <b>Cell type-specific annotation and fine mapping of variants associated with brain disorders.</b></em> Common genetic variants confer susceptibility to a large number of complex brain disorders. Given that such variants predominantly localize in non-coding regions of the human genome, there is a significant challenge to predict and characterize their functional consequences. More importantly, available context-free methods do not take into account tissue/cell specificity of the diseases, limiting the ability to predict the functional consequences of common variants on brain disorders. In this study, we introduce a comparative multi-step pipeline to investigate the relative effectiveness of context-specific and context-free approaches to prioritize disease causal variants. More can be found <a href="https://www.frontiersin.org/articles/10.3389/fgene.2020.575928/abstract"><span class="label"> here.</span></a></div> </p>
								  
								  <p> <div class="col-3"><span class="image left"><img src="images/SR.jpg"   alt="" /></span> K.E. Borgmann-Winter, K. Wang, S. Bandyopadhyay, <span style="text-decoration: underline;">A. Doostparast Torshizi</span>, I. Blair, C.G. Hahn,<em><b> The proteome and its dynamics: A missing piece for integrative multi-omics in schizophrenia.</b></em> The complex and heterogeneous pathophysiology of schizophrenia can be deconstructed by integration of large-scale datasets encompassing genes through behavioral phenotypes. Genome-wide datasets are now available for genetic, epigenetic and transcriptomic variations in schizophrenia, which are then analyzed by newly devised systems biology algorithms. A missing piece, however, is the inclusion of information on the proteome and its dynamics in schizophrenia. Proteomics has lagged behind omics of the genome, transcriptome and epigenome since analytic platforms of proteins were previously not as robust as those for nucleic acids. In recent years, however, there has been an unprecedented progress in the instrumentation (liquid chromatography (LC) and mass spectrometry (MS)), experimental paradigms, and bioinformatics of the proteome. Large-scale analyses of the schizophrenia proteome are now possible and ought to be pursued vigorously and integrated with other omics results. With that in our view, we review proteomics studies that have been conducted in schizophrenia to date, present a summary of methodological innovations of recent years in MS based proteomics and the power of new generation proteomics, and propose how such data can be analyzed and integrated with other omics results (<a href="https://www.sciencedirect.com/science/article/abs/pii/S0920996419303032"><span class="label">Publication URL</span></a>.)</div></p>   
								 
								 <p> <div class="col-3"><span class="image left"><img src="images/drug.JPG"  alt="" /></span><span style="text-decoration: underline;">A. Doostparast Torshizi</span>, K. Wang,<em> <b>Next-generation sequencing in drug development: target identification and genetically stratified clinical trials.</b></em> Next-generation sequencing (NGS) enabled high-throughput analysis of genotype-phenotype relationships on human populations, ushering in a new era of genetics-informed drug development. The year 2017 was remarkable, with the first US Food and Drug Administration (FDA)-approved gene therapy for cancer (Kymriah<sup>&#153</sup>) and for inherited diseases (LUXTURNA<sup>&#153</sup>), the first multiplex NGS panel for companion diagnostics (MSK-IMPACT), and the first drug targeting a genetic signature rather than a disease (Keytruda<sup>&reg</sup>). We envision that population-scale NGS with paired electronic health records (EHRs) will become a routine measure in the drug development process for the identification of novel drug targets, and that genetically stratified clinical trials may be widely adopted to improve power in precision medicine-guided drug development. More can be found <a href="https://www.sciencedirect.com/science/article/pii/S1359644618300400"><span class="label"> here.</span></a></div> </p>
					              
								 <p> <div class="col-3"><span class="image left"><img src="images/Jamia.png"  alt="" /></span><span style="text-decoration: underline;">A. Doostparast Torshizi</span>, L. Petzold, <em><b>Graph-based semi-supervised learning with genomic data integration using condition-responsive genes applied to phenotype classification,</b></em><span style="text-decoration: underline;"> (Featured Article & Editor's Choice)</span>. Data integration methods that combine data from different molecular levels such as genome, epigenome, transcriptome, etc., have received a great deal of interest in the past few years. It has been demonstrated that the synergistic effects of different biological data types can boost learning capabilities and lead to a better understanding of the underlying interactions among molecular levels. In this paper we present a graph-based semi-supervised classification algorithm that incorporates latent biological knowledge in the form of biological pathways with gene expression and DNA methylation data. The process of graph construction from biological pathways is based on detecting condition-responsive genes, where 3 sets of genes are finally extracted: all condition responsive genes, high-frequency condition-responsive genes, and P-value–filtered genes. More can be found <a href="https://academic.oup.com/jamia/article/25/1/99/3826530"><span class="label">here.</span></a></div> </p>
						
								 <p> <div class="col-3"><span class="image left"><img src="images/IEEE.JPG"  alt="" /></span><span style="text-decoration: underline;">A. Doostparast Torshizi</span>, L. Petzold,<em> <b> Sparse Pathway-Induced Dynamic Network Biomarker Discovery for Early Warning Signal Detection in Complex Diseases. </b></em>In many complex diseases, the transition process from the healthy stage to the catastrophic stage does not occur gradually. Recent studies indicate that the initiation and progression of such diseases are comprised of three steps including healthy stage, pre-disease stage, and disease stage. It has been demonstrated that a certain set of trajectories can be observed in the genetic signatures at the molecular level, which might be used to detect the pre-disease stage and to take necessary medical interventions. In this paper, we propose two optimization-based algorithms for extracting the dynamic network biomarkers responsible for catastrophic transition into the disease stage, and to open new horizons to reverse the disease progression at an early stage through pinpointing molecular signatures provided by high-throughput microarray data. The first algorithm relies on meta-heuristic intelligent search to characterize dynamic network biomarkers represented as a complete graph. The second algorithm induces sparsity on the adjacency matrix of the genes by taking into account the biological signaling and metabolic pathways, since not all the genes in the ineractome are biologically linked. Comprehensive numerical and meta-analytical experiments verify the effectiveness of the results of the proposed approaches in terms of network size, biological meaningfulness, and verifiability. More can be found <a href="https://ieeexplore.ieee.org/abstract/document/7887681"><span class="label">here.</span></a></div> </p>
								 
								 <p> <div class="col-3"><span class="image left"><img src="images/Neuro.jpg"  alt="" /></span><span style="text-decoration: underline;">A. Doostparast Torshizi</span>, L. Petzold, M. Cohen,<em> <b> Multivariate soft repulsive system identification for constructing rule-based classification systems: Application to trauma clinical data. </b></em> Rule-based classification systems constructed upon linguistic terms in the antecedent and consequent of the rules lack sufficient generalization capabilities. This paper proposes a new multivariate fuzzy system identification algorithm to design binary rule-based classification structures through making use of the repulsive forces between the cluster prototypes of different class labels. This approach is coupled with the potential discrimination power of each dimension in the feature space to increase the generalization potential. To address this issue, first the multivariate variant of a newly proposed soft clustering algorithm along with its mathematical foundations is proposed. Next, the discriminatory power of each individual feature is computed, using the multivariate membership values in the proposed clustering algorithm to achieve the most accurate firing degree in each rule. The main advantage of this method is to handle unbalanced datasets yielding superior true positive measure while keeping the false positive rate low enough to avoid the natural bias toward class labels containing larger number of training samples. To validate the proposed approaches, a series of numerical experiments on publicly available datasets and a real clinical dataset collected by our team were conducted. Simulation results demonstrated achievement of the primary goals of this research. More can be found <a href="https://www.sciencedirect.com/science/article/pii/S0925231217305581"><span class="label">here.</span></a></div> </p>
								 
								 <p> <div class="col-3"><span class="image left"><img src="images/Know.jpg"  alt="" /></span><span style="text-decoration: underline;">A. Doostparast Torshizi</span>, M.H. Fazel Zarandi,<em> <b> A new cluster validity measure based on general type-2 fuzzy sets: Application in gene expression data clustering.</b></em> As a widespread pattern recognition technique, clustering has been widely used in various disciplines including: science, engineering, medicine, etc. One the latest progresses in this field is introduction of general type-2 fuzzy sets and the new clustering method represented on its basis called general type-2 fuzzy c-means. In this paper, the aim is to develop a robust and accurate similarity measure between general type-2 fuzzy sets. Utilizing philosophy behind this developed similarity measure, the first exclusively developed general type-2 fuzzy cluster validity index will be proposed to be used for finding the optimal number of clusters through using general type-2 fuzzy c-means. To verify quality of the proposed approaches, several heavy computations have been conducted on artificial datasets and also real gene expression datasets. Numerical comparisons reveal robustness and quality of the proposed approach compared to several similar approaches in the literature. More can be found <a href="https://www.sciencedirect.com/science/article/abs/pii/S0950705114001129"><span class="label"> here.</span></a></div> </p>
								 
								 </header>
												
							</section>	

							   <section id="news" class="main ">
                              <header class="major spacial">
									<h2>News & Media</h2>
								<p>I usually share my career-related news here</a>.</p>
							
											 <ul class="alt"> 
                                                                                          <li> <strong> July 2024: I am honored to receive the Institutional Leadership Award from JnJ Innovative Medicine.</strong></li>
											 <li> <strong> August 2024: We have a poster accepted to <a href="https://www.ashg.org/meetings/2024meeting/"><span class="label">the 2024 American Society of Human Genetics (ASHG) Annual Meeting</span></a> in Denver, Colorado. This poster is based on our work on proteomic biomarker discovery using machine learning on Parkinson's disease.</strong> <b>This work received the Reviewer's Choice Award. </b> </li>
											<li> <strong> August 2023: We have two posters accepted to <a href="https://www.ashg.org/meetings/2023meeting/"><span class="label">the 2023 American Society of Human Genetics (ASHG) Annual Meeting</span></a> in Washington DC. These posters are based on our work on population-scale proteomics data from the UKB-PPP consortium for unsupervised subtyping of Parkinson's patients as well as comprehensive integration of protein-disease association with Mendelian randomization to reveal potntial drug targets and disease biomarkers.</strong></li>	 
											<li> <strong> July 2023: Our work titled "Shared genetic architecture between Alzheimer’s disease and Parkinson's disease" is accepted for presentation at Keystone Symposia: Neurodegeneration: New biology guiding the next generation of therapeutic development, Canada 2023.</strong></li>
											<li> <strong> October 2022: My work has received the reviewer's recognition at <a href="https://www.ashg.org/meetings/2022-annual-meeting/"><span class="label">the 2022 American Society of Human Genetics (ASHG) Annual Meeting</span></a> in Los Angeles.</strong></li>
											 <li> <strong> September 2022: My work is now published in <a href="https://www.nature.com/articles/s41540-022-00243-8"><span class="label">npj Systems Biology and Applications</span></a>.</strong></li>
											 <li> <strong> June 2021: My presentation at Caltech Bioinformatics Seminars:</strong></li>
											 <style>.embed-container { position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; } .embed-container iframe, .embed-container object, .embed-container embed { position: absolute; top: 0; left: 0; width: 100%; height: 100%; }</style><div class='embed-container'> <iframe width="560" height="315" src="https://www.youtube.com/embed/vIOmaYbsaHE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></div>
											 
											 
											 
											 <li> <strong> June 2021: Our work on Huntington's disease is accepted for publication in Genes and Diseases.</span></a> </strong></li>
											 <li> <strong> June 2021: The CellR paper is accepted for publication in NAR Genomics and Bioinformatics.</span></a> </strong></li>
											  <li> <strong> June 2021: I will be giving a talk at Caltech Bioinformatics Seminar.</span></a> </strong></li>
											  <li> <strong> March 2021: I will be giving a faculty seminar at the Children's Hospital of Philadelphia.</span></a> </strong></li>
											 <li> <strong> Nov. 2020: Our paper titled "Cell type-specific annotation and fine mapping of variants associated with brain disorders" is accepted for publication in <a href="https://www.frontiersin.org/articles/10.3389/fgene.2020.575928/abstract"><span class="label">Frontiers of Genetics</span></a>.</strong></li>
											 <li> <strong> July 2020: My abstract is accepted for presentation at <a href="https://www.ashg.org/meetings/2020meeting/"><span class="label">ASHG 2020</span></a> in October.</strong></li>
											 <li> <strong> July 2020: Our paper titled "Cell type-specific proteogenomic signal diffusion for integrating multi-omics data predicts novel schizophrenia risk genes" is accepted for publication in <a href="https://www.cell.com/patterns/home"><span class="label">Patterns</span></a>.</strong></li>
											 <li> <strong> Oct. 2019: I am honored to be selected as the <a href="https://training.research.chop.edu/prospective-trainees/1183-october-featured-chop-research-trainee-abolfazl-doostparast-phd.html"><span class="label">Featured Research Trainee</span></a> at the Children's Hospital of Philadelphia.</strong></li>
											 <li> <strong> Sep. 2019: Press coverage of our work (in Farsi) <a href="https://www.isna.ir/news/98062612842/%DA%98%D9%86-%D8%AA%D9%86%D8%B8%DB%8C%D9%85-%DA%A9%D9%86%D9%86%D8%AF%D9%87-%D8%A7%D8%B3%DA%A9%DB%8C%D8%B2%D9%88%D9%81%D8%B1%D9%86%DB%8C-%D8%A8%D8%A7-%D9%87%D9%85%DA%A9%D8%A7%D8%B1%DB%8C-%D8%AF%D8%A7%D9%86%D8%B4%D9%85%D9%86%D8%AF-%D8%A7%DB%8C%D8%B1%D8%A7%D9%86%DB%8C-%DA%A9%D8%B4%D9%81-%D8%B4%D8%AF"><span class="label">ISNA</span></a>, <a href="https://www.yjc.ir/fa/news/7074246/%DA%A9%D8%B4%D9%81-%DA%98%D9%86-%D8%AA%D9%86%D8%B8%DB%8C%D9%85-%DA%A9%D9%86%D9%86%D8%AF%D9%87-%D8%A8%DB%8C%D9%85%D8%A7%D8%B1%DB%8C-%D8%A7%D8%B3%DA%A9%DB%8C%D8%B2%D9%88%D9%81%D8%B1%D9%86%DB%8C"><span class="label">YJC</span></a>, <a href="https://www.irinn.ir/fa/news/719556/%DA%A9%D8%B4%D9%81-%DA%98%D9%86-%D8%AA%D9%86%D8%B8%DB%8C%D9%85-%DA%A9%D9%86%D9%86%D8%AF%D9%87-%D8%A8%DB%8C%D9%85%D8%A7%D8%B1%DB%8C-%D8%A7%D8%B3%DA%A9%DB%8C%D8%B2%D9%88%D9%81%D8%B1%D9%86%DB%8C"><span class="label">IRINN</span></a>, <a href="https://ana.ir/fa/news/10/415885/%D8%AF%D8%A7%D9%86%D8%B4%D9%85%D9%86%D8%AF%D8%A7%D9%86-%DA%98%D9%86-%D8%AA%D9%86%D8%B8%DB%8C%D9%85%E2%80%8C%DA%A9%D9%86%D9%86%D8%AF%D9%87-%D8%A7%D8%B5%D9%84%DB%8C-%D8%A8%DB%8C%D9%85%D8%A7%D8%B1%DB%8C-%D8%B4%DB%8C%D8%B2%D9%88%D9%81%D8%B1%D9%86%DB%8C-%D8%B1%D8%A7-%DA%A9%D8%B4%D9%81-%DA%A9%D8%B1%D8%AF%D9%86%D8%AF"><span class="label">ANA</span></a>, <a href="https://etemadonline.com/content/342266/%DA%98%D9%86-%D8%AA%D9%86%D8%B8%DB%8C%D9%85-%DA%A9%D9%86%D9%86%D8%AF%D9%87-%D8%A7%D8%B3%DA%A9%DB%8C%D8%B2%D9%88%D9%81%D8%B1%D9%86%DB%8C-%D8%A8%D8%A7-%D9%87%D9%85%DA%A9%D8%A7%D8%B1%DB%8C-%D8%AF%D8%A7%D9%86%D8%B4%D9%85%D9%86%D8%AF-%D8%A7%DB%8C%D8%B1%D8%A7%D9%86%DB%8C-%DA%A9%D8%B4%D9%81-%D8%B4%D8%AF"><span class="label">Etemad News</span></a>, <a href="https://www.tasnimnews.com/fa/news/1398/06/27/2098882/%DA%A9%D8%B4%D9%81-%DA%98%D9%86-%D8%A8%DB%8C%D9%85%D8%A7%D8%B1%DB%8C-%D8%A7%D8%B3%DA%A9%DB%8C%D8%B2%D9%88%D9%81%D8%B1%D9%86%DB%8C-%D8%A8%D8%A7-%D9%87%D9%85%DA%A9%D8%A7%D8%B1%DB%8C-%D8%AF%D8%A7%D9%86%D8%B4%D9%85%D9%86%D8%AF-%D8%A7%DB%8C%D8%B1%D8%A7%D9%86%DB%8C"><span class="label"> Tasnim News</span></a>, <a href="https://www.mehrnews.com/news/4721764/%DA%98%D9%86-%D8%AA%D9%86%D8%B8%DB%8C%D9%85-%DA%A9%D9%86%D9%86%D8%AF%D9%87-%D8%A8%DB%8C%D9%85%D8%A7%D8%B1%DB%8C-%D8%A7%D8%B3%DA%A9%DB%8C%D8%B2%D9%88%D9%81%D8%B1%D9%86%DB%8C-%DA%A9%D8%B4%D9%81-%D8%B4%D8%AF"><span class="label">Mehr News</span></a>, <a href="https://www.nabaapress.ir/%D8%A7%D9%82%D8%AA%D8%B5%D8%A7%D8%AF%DB%8C/%D9%81%D9%86%D8%A7%D9%88%D8%B1%DB%8C-%D8%A7%D8%B7%D9%84%D8%A7%D8%B9%D8%A7%D8%AA-%D8%A7%DB%8C%D9%86%D8%AA%D8%B1%D9%86%D8%AA-%D9%85%D9%88%D8%A8%D8%A7%DB%8C%D9%84/%DA%98%D9%86-%D8%AA%D9%86%D8%B8%DB%8C%D9%85-%DA%A9%D9%86%D9%86%D8%AF%D9%87-%D8%A8%DB%8C%D9%85%D8%A7%D8%B1%DB%8C-%D8%A7%D8%B3%DA%A9%DB%8C%D8%B2%D9%88%D9%81%D8%B1%D9%86%DB%8C-%DA%A9%D8%B4%D9%81-%D8%B4%D8%AF"><span class="label">Nabaa Press</span></a>, <a href="http://fanavarimag.ir/1398/06/27/%DA%98%D9%86-%D8%AA%D9%86%D8%B8%DB%8C%D9%85-%DA%A9%D9%86%D9%86%D8%AF%D9%87-%D8%A8%DB%8C%D9%85%D8%A7%D8%B1%DB%8C-%D8%A7%D8%B3%DA%A9%DB%8C%D8%B2%D9%88%D9%81%D8%B1%D9%86%DB%8C-%DA%A9%D8%B4%D9%81-%D8%B4%D8%AF"><span class="label">Fanavar Magazine</span></a>, <a href="https://soraya.news/fa/news-details/49754/"><span class="label">Soraya News</span></a>, <a href="https://khabari.org/article/2599866/%DA%A9%D8%B4%D9%81-%DA%98%D9%86-%D8%A8%DB%8C%D9%85%D8%A7%D8%B1%DB%8C-%D8%A7%D8%B3%DA%A9%DB%8C%D8%B2%D9%88%D9%81%D8%B1%D9%86%DB%8C-%D8%A8%D8%A7-%D9%87%D9%85%DA%A9%D8%A7%D8%B1%DB%8C-%D8%AF%D8%A7%D9%86%D8%B4%D9%85%D9%86%D8%AF-%D8%A7%DB%8C%D8%B1%D8%A7%D9%86%DB%8C"><span class="label">Khabari.org</span></a>, <a href="http://www.hamrahanekhabar.com/news/428772/%DA%98%D9%86-%D8%AA%D9%86%D8%B8%DB%8C%D9%85-%DA%A9%D9%86%D9%86%D8%AF%D9%87-%D8%A7%D8%B3%DA%A9%DB%8C%D8%B2%D9%88%D9%81%D8%B1%D9%86%DB%8C-%D8%A8%D8%A7-%D9%87%D9%85%DA%A9%D8%A7%D8%B1%DB%8C-%D8%AF%D8%A7%D9%86%D8%B4%D9%85%D9%86%D8%AF-%D8%A7%DB%8C%D8%B1%D8%A7%D9%86%DB%8C-%DA%A9%D8%B4%D9%81-%D8%B4%D8%AF"><span class="label">Hamrahane Khabar</span></a>.</strong></li>
											 
											  <li> <strong> Sep. 2019: My poster on a new muti-omics data integration method is accepted in <a href="https://www.mabc2019.org/"><span class="label">2019 MidAtlantic Bioinformatics Conference</span></a>.</strong></li>
											 <li> <strong> Sep. 2019: Coverage of our findings on <a href="https://www.eurekalert.org/pub_releases/2019-09/chop-sig091019.php"><span class="label">EurekAlert AAAS</span></a>.</strong></li>
											  <span class="center"><img src="images/EA.png" style="max-width:100%"    /></span>
											 <li> <strong> Sep. 2019: Our findings on schizophrenia is covered by <a href="https://www.genengnews.com/news/gene-acts-as-master-regulator-of-schizophrenia/"><span class="label">Genetic Engineering & Biotechnology News</span></a>.</strong></li>
											 <span class="center"><img src="images/News.jpg" style="max-width:100%"    /></span>
											  <li> <strong> Sep. 2019: Coverage of our findings on <a href="https://www.genomeweb.com/scan/science-finds-support-gene-drives-depends-species-targeted-ability-control-drive-more#.XYOST2YpBaQ"><span class="label">Genome Web</span></a>.</strong></li>
											  <span class="center"><img src="images/GW.png" style="max-width:100%"    /></span>
											 <li> <strong> Sep. 2019: Coverage of our findings on <a href="https://www.sciencedaily.com/releases/2019/09/190911142806.htm"><span class="label">ScienceDaily</span></a>.</strong></li>
											  <span class="center"><img src="images/SD.png" style="max-width:100%"    /></span>
											  <li> <strong> Sep. 2019: <a href="https://www.clinicalomics.com/topics/informatics-topic/chop-researchers-id-schizophrenia-master-regulator-gene/"><span class="label">CHOP Researchers ID Schizophrenia Master Regulator Gene</span></a>.</strong></li>
											   <span class="center"><img src="images/CO.jpg" style="max-width:100%"    /></span>
											   <li> <strong> Sep. 2019: Coverage on <a href="https://newatlas.com/medical/master-regulator-gene-for-schizophrenia-discovered/"><span class="label">New Atlas</span></a>.</strong></li>
											 <span class="center"><img src="images/NA.jfif" style="max-width:100%"    /></span>
											  <li> <strong> Sep. 2019: <a href="https://www.news-medical.net/news/20190912/Scientists-discover-major-regulator-of-schizophrenia-that-is-active-during-early-development.aspx"><span class="label">Scientists discover major regulator of schizophrenia that is active during early development</span></a>.</strong></li>
											  <span class="center"><img src="images/NM.jpg" style="max-width:100%"    /></span>
											  <li> <strong> Sep. 2019: Our findings on schizophrenia is covered by <a href="https://medicalxpress.com/news/2019-09-scientists-gene-master-schizophrenia.html"><span class="label">Medical XPress</span></a>.</strong></li>
											    <span class="center"><img src="images/MX.jpg" style="max-width:100%"    /></span>
												<li> <strong> Sep. 2019: Coverage of our findings on <a href="https://notiziescientifiche.it/scienziati-scoprono-gene-regolatore-della-schizofrenia/"><span class="label">Notizie Scientifiche (Italian news outlet)</span></a>.</strong></li>
											  <span class="center"><img src="images/NS.jpg" style="max-width:100%"    /></span>
											  	<li> <strong> Sep. 2019: Schizophrenia: Master Regulator Discovered on <a href="https://www.medindia.net/news/schizophrenia-master-regulator-discovered-190328-1.htm"><span class="label">Med India (Indian news outlet)</span></a>.</strong></li>
											  <span class="center"><img src="images/MI.jpg" style="max-width:100%"    /></span>
											 <li> <strong> Sep. 2019: A short video on our latest paper appeared in Science Advances:</strong></li>
											 <style>.embed-container { position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; } .embed-container iframe, .embed-container object, .embed-container embed { position: absolute; top: 0; left: 0; width: 100%; height: 100%; }</style><div class='embed-container'><iframe src='https://www.youtube.com/embed/g41kyiypHxU' frameborder='0' allowfullscreen></iframe></div>
											 <li> <strong> August 2019: My paper published in <a href="https://academic.oup.com/jamia/article/25/1/99/3826530"><span class="label">The Journal of American Medical Informatics Association</span></a> is selected as the best paper in <a href="https://www.thieme-connect.com/products/ejournals/journal/10.1055/s-00034612"><span class="label">IMIA Yearbook of Medical Informatics 2019</span></a> in the section 'Bioinformatics and Translational Informatics'.</strong></li>
											  <span class="center"><img src="images/Capture.JPG" style="max-width:100%"    /></span>
											<li> <strong> July 2019: My abstract is accepted for presentation at <a href="https://www.ashg.org/2019meeting/"><span class="label">ASHG 2019</span></a> in Houston in October.</strong></li>
											<li> <strong> July 2019: Our review paper on proteomics and its dynamics for integrative multi-omics in schizophrenia will appear in <a href="https://www.journals.elsevier.com/schizophrenia-research"><span class="label">Schizophrenia Research</span></a>.</strong></li>		
											<li> <strong> June 2019: Excited to announce that our paper has been accepted for publication in <a href="https://advances.sciencemag.org/"><span class="label"> Science Advances</span></a>.</strong></li>								
											<li> <strong> June 2019: Presented my findings at the <a href="https://www.med.upenn.edu/sages/"><span class="label"> 2019 Symposium on Advances in Genomics, Epidemiology, and Statistics (SAGES)</span></a> at <a href="https://www.med.upenn.edu/"><span class="label"> Perelman School of Medicine at the University of Pennsylvania</span></a>.</strong></li>						
											<li> <strong> May 2019: Presented a poster at the <a href="http://upibi.org/event/informatics-day-2019/"><span class="label"> 2019 Informatics Day</span></a> at <a href="http://upibi.org/"><span class="label"> Penn Institute for Biomedical Informatics</span></a>.</strong></li>								
											<li> <strong> May 2019: Presented a poster of my findings on transcriptional curcuitry of schizophrenia at the <a href="https://pdss.research.chop.edu/"><span class="label"> 2019 CHOP Research Institute Scientific Symposium</span></a>.</strong></li>						
										    <li> <strong> May 2019: Honored to be awarded the 2019 Alavi-Dabiri Postdoctoral Fellowship Award from the <a href="https://iddrc.research.chop.edu/training_programs_dabiri.php"><span class="label"> Intellectual and Disabilities Research Center (IDDRC)</span></a> at Penn/CHOP.</strong></li>	
											<li> <strong> Mar. 2019: Our proposal "Genome-wide structural variation detection in acute lymphocytic leukemia using long-read sequencing technology" got funded through Roberts Rapid Grants at CHOP/Penn.</strong></li>										
                                            <li> <strong> Feb. 2019: Our  <a href="https://www.frontiersin.org/articles/10.3389/fnana.2019.00023/abstract"><span class="label"> collaborative research</span></a> investigating the functional impact of SHANK3 deletions on autism spectrum disorders using human induced pluripotent stem cell (hiPSC)-derived neurons is out in <em> frontiers in Neuroanatomy</em>.</strong></li>										
										    <li> <strong> Dec. 2018: My <a href="https://bmcresnotes.biomedcentral.com/articles/10.1186/s13104-018-3594-0"><span class="label"> findings</span></a> on network regulators of autism spectrum disorders are published in  <em>  BMC Research Notes</em>.</strong></li>
											<li> <strong> Jun. 2018: I am honored to serve as the <a href="http://icibm2018.zhaobioinfo.org/Organization.htm"><span class="label"> Organizing Committee Member and Chair </span></a> of the <a href="http://icibm2018.zhaobioinfo.org/"><span class="label"> International Conference on Intelligent Biology and Medicine (ICIBM 2018) </span></a> held in Los Angeles, USA.</strong></li>
											<li> <strong> Apr. 2018: I am excited to have <a href="https://www.sciencedirect.com/science/article/pii/S1359644618300400"><span class="label">  my review</span></a> on the applications of next generation sequencing technologies in drug development and design of stratified clinical trials published in <em>  Drug Discovery Today</em>.</strong></li>
											<li> <strong> Aug. 2017: I successfully hold the first<a href="https://muii.missouri.edu/bibm2017/workshops#content"><span class="label"> International Workshop on Translational Bioinformatics in Precision Medicine (TBPM),</span></a> in conjunction with the <a href="https://muii.missouri.edu/bibm2017/"><span class="label">2017 IEEE International Conference on Bioinformatics and Biomedicine (IEEE BIBM 2017) </span></a> held at Kansas City, MO, USA.</strong></li>
											<li> <strong> Aug. 2017: I am excited to have <a href="https://www.nature.com/articles/s41598-017-15221-y"><span class="label">  my research</span></a> on decovolution of transcriptional networks aimed at uncovering the master regulators of PTSD published in <em>  Nature Scientific Reports</em>.</strong></li>
											<li> <strong> Jul. 2017: I gave a talk on <a href="https://ghmicr.jnu.edu.cn/08/4c/c9536a264268/page.htm"><span class="label"> Reverse engineering of transcriptional networks in psychiatric disorders</span></a> at Jinan University, China.         </strong></li>
											<li> <strong> Apr. 2017: Excited to announce that I will be holding a workshop in conjunction with the 2017 IEEE International Conference on Bioinformatics and Biomedicine (IEEE BIBM 2017) to be held in Kansas City, MO, USA. </strong></li>
											<li> <strong> Feb. 2017: My <a href="https://www.sciencedirect.com/science/article/pii/S0925231217305581"><span class="label"> research</span></a>  on developing a multivariate soft repulsive systems identification for patient stratification applied to trauma clinical data is published in <em> Neurocomputing  </em>. </strong></li>
											<li> <strong> Dec. 2016: Excited to have <a href="https://ieeexplore.ieee.org/abstract/document/7887681"><span class="label"> my paper</span></a> on sparse pathway-induced dynamic network biomarker discovery in complex diseases published in <em> IEEE/ACM Transactions on Computational Biology and Bioinformatics  </em>. </strong></li>
											<li> <strong> Oct. 2016: Honored to have <a href="https://academic.oup.com/jamia/article/25/1/99/3826530"><span class="label"> my paper</span></a> published as the Editor's Choice in the <em>Journal of the American Medical Informatics Association </em>. </strong></li>
											<li> <strong> Oct. 2016: I presented my work on PTSD at the<a href="https://www.ucsf.edu/"><span class="label"> University of California, San Francisco </span></a>. </strong></li>
											<li> <strong> Sep. 2016: I gave a seminar at <a href="https://systemsbiology.org/"><span class="label"> Institute for Systems Biology (ISB) </span></a>, Seattle, USA. </strong></li>
											<li> <strong> Jan. 2016: I gave a talk at <a href="https://www.seas.harvard.edu/"><span class="label"> Harvard John A. Paulson School of Engineering and Applied Sciences </span></a>. </strong></li>
											<li> <strong> Dec. 2015: I gave a seminar at <a href="https://frederick.cancer.gov/"><span class="label">Fredrick National Lab </span></a>,   Washington DC, USA. </strong></li>
											<li> <strong> Nov. 2015: I presented  <a href="https://ieeexplore.ieee.org/abstract/document/7359795"><span class="label">my paper </span></a> at <a href="https://cci.drexel.edu/ieeebibm/bibm2015/"><span class="label">2015 IEEE International Conference on Bioinformatics and Biomedicine (IEEE BIBM 2015) </span></a>,  Fredrick, MD, USA. </strong></li>
											<li> <strong> Apr. 2015: I successfully defended my PhD suma cum laude. </strong></li>
											<li> <strong> Feb. 2015: My <a href="https://www.sciencedirect.com/science/article/pii/S1568494614001963"><span class="label">comprehensive literature review </span></a> on type-reduction of type-2 fuzzy sets is published in <em>Soft Computing</em>. </strong></li>
											<li> <strong> Sep. 2014: My research on developing <a href="https://www.sciencedirect.com/science/article/pii/S0020025514003028"><span class="label"> Hierarchical Collapsing Method</span></a> for direct defuzzification of general type-2 fuzzy sets is out in <em>Information Sciences</em>. </strong></li>
											<li> <strong> Jul. 2014: My novel <a href="https://www.sciencedirect.com/science/article/abs/pii/S0950705114001129"><span class="label">cluster validity measure </span></a> for gene expression segmentation is published in <em>Knowledge-Based Systems</em>. </strong></li>
											<li> <strong> Jul. 2014: I received the best paper award at  <a href="http://21stcenturywiener.org/nafips/"><span class="label">2014 IEEE Conference on Norbert Wiener in the 21st Century (21CW), </span></a> in conjunction with NAFIPS 2014, held in Boston, USA.</strong></li>
											<li> <strong> Jan. 2014: My <a href="https://www.sciencedirect.com/science/article/pii/S0169260713003295"><span class="label">work </span></a> on developing an intelligent system for treatment recommendation in individuals with bening prostatic hyperplasia is published in  <em>Computer Methods and Programs in Biomedicine</em>.</strong></li>
                                            <li> <strong> May. 2013: My research on developing an indirect data-driven approach for system identification is published in <a href="https://www.sciencedirect.com/science/article/pii/S002002551200802X"><span class="label">Information Sciences</span></a>.</strong></li>											
											<li> <strong> Aug. 2012: I presented <a href="https://ieeexplore.ieee.org/document/6291067"><span class="label">my paper </span></a> at  <a href="https://sites.ualberta.ca/~reformat/nafips2012/index.html"><span class="label">North American Information Processing Society Conference (IEEE-NAFIPS) </span></a> at UC Berkeley.</strong></li>
											<li> <strong> Apr. 2012: My papers got accepted for oral presentation at  <a href="http://www.ims.sakarya.edu.tr/"><span class="label">8th International Symposium on Intelligent and Manufacturing Systems (IMS 2012)</span></a> to be held in Antalya, Turkey.</strong></li>
											<li> <strong> Apr. 2012: My paper got accepted for oral presentation at  <a href="https://international-aset.com/ICECS2012/"><span class="label">2012 International Conference on Electrical and Computer Systems (ICECS'12)</span></a> to be held in Ottawa, Canada.</strong></li>
											<li> <strong> Nov. 2011: I joined Computational Intelligence Lab led by <a href="http://industrial.aut.ac.ir/autcms/people/verticalPagesAjax/professorHomePage.htm?url=zarandi&depurl=industrial-engineering&lang=en&cid=759"><span class="label">Professor Mohammad Hossein Fazel Zarandi</span></a> as a graduate research assistant.</strong></li>
											<li> <strong> Jun. 2011: I am proud t announce that I have been admitted to the PhD program at <a href="http://aut.ac.ir/aut/"><span class="label">Amirkabir University of Technology (Tehran Polytechnic).</span></a></strong></li>
											<li> <strong> April. 2011: I received the <a href="https://iut.ac.ir/en"> Isfahan University of Technology (IUT)</a> Presidential Award.</strong></li>
											<li> <strong> Jan. 2011: I successfully defended my Master's thesis with the highest honors.</strong></li>
											<li> <strong> Dec. 2010: My paper just got published in the<a href="https://www.hindawi.com/journals/afs/2012/957697/"> Journal of Industrial Engineering</a>.</strong></li>
											<li> <strong> Nov. 2010: My paper just got published in <a href="https://www.hindawi.com/journals/afs/2012/957697/"> Advances in Fuzzy Systems</a>.</strong></li>
											<li> <strong> Nov. 2010: My paper just got published in <a href="https://www.sciencedirect.com/science/article/pii/S0957417411001217"> Expert Systems with Applications</a>.</strong></li>
											<li> <strong> Sep. 2010: I presented at Iranian Fuzzy Systems annual conference.</strong></li>
									        <li> <strong> Oct. 2009: I received a two year fellowship as the top MS student.</strong></li>
											<li> <strong> Sep. 2009: Honored to be among the recipients of a fully funded 3-week summer school by German Academic Exchange Service (DAAD). </strong></li>

                                             </ul>

																				
							</section>
								
							<section id="fund" class="main ">
								<header class="major">
									<h2>Funding & Awards</h2>
								</header>
                             <h2 class="major">Funding</h2>
									    <ul class="alt">
                                            <li> <strong> Mar. 2019: Roberts Rapid Grant for the project "Genome-wide structural variation detection in acute lymphocytic leukemia using long-read sequencing technology" ($10,000, Role: Senior/Key Personnel).</strong></li>																				  
                                 </ul>
									<h2 class="major">Honors and Awards</h2>
									<ul class="alt">
									<li> <strong> Oct. 2022: I received reviewer's recognition at the 2022 ASHG annual meeting in LA.</strong></li>
									<li> <strong> Oct. 2019: <a href="https://training.research.chop.edu/prospective-trainees/1183-october-featured-chop-research-trainee-abolfazl-doostparast-phd.html"><span class="label">Featured Research Trainee</span></a>, Children's Hospital of Philadelphia.</strong></li>
									<li> <strong> May 2019: <a href="https://iddrc.research.chop.edu/training_programs_dabiri.php"> Alavi-Dabiri Postdoctoral Fellowship Award</a>, Perelman School of Medicine, University of Pennsylvania, The Children's Hospital of Philadelphia.</strong></li>
									 <li> <strong> Dec. 2018: <em> Distinguished Research Trainee Award Nominee</em>, The Children's Hospital of Philadelphia.</strong></li>
                                     <li> <strong> Apr. 2015: <em> Excellent PhD Dissertation Award</em>, Amirkabir University of Technology (Tehran Polytechnic), Iran.</strong></li>
									 <li> <strong> Nov. 2014: <em> Recognized Reviewer Award</em>, Elsevier Publishers.</strong></li>	
                                     <li> <strong> Aug. 2014: <em> Best Student Paper Award</em>, IEEE-North American Fuzzy Information Processing Society Conference (NAFIPS), Boston, USA.</strong></li>	
									 <li> <strong> Sep. 2011: <em> Admission with Honor as an Exceptional Graduate Student</em> to Doctoral Program, Amirkabir University of Technology (Tehran Polytechnic), Iran.</strong></li>	
									 <li> <strong> Jan. 2011: <em> Top-ranked MS Student Award</em>, Isfahan University of Technology.</strong></li>
                                     <li> <strong> Sep. 2009: <em> Fellowship and Summer School Grant</em>, German Academic Exchange Service (DAAD).</strong></li>		
                                     <li> <strong> Jun. 2009 - Jun. 2011: <em> Isfahan University of Technology Fellowship</em>, Ranked First among all of the graduate students, Isfahan University of Technology, Iran.</strong></li>											 

                                      </ul>

							</section>
								
							<section id="servis" class="main ">
								<header class="major">
									<h2>Service</h2>
								</header>
                              <div class="content span-7" style="margin-left:20px" style="margin-top:30px">
									    <ul class="alt">
									<li> <strong> <em> Program Committee Member</em>, International Conference on Intelligent Biology and Medicine (ICIBM 2020), Philadelphia, PA, USA.</strong></li>	 
									<li> <strong> <em> Organizing Committee Member and Chair</em>, International Conference on Intelligent Biology and Medicine (ICIBM 2018), Los Angeles, CA, USA.</strong></li>
									<li> <strong> <em> Workshop Organizer and Chair</em>, International Workshop on Translational Bioinformatics in Precision Medicine (TBPM 2017), In Conjunction with 2017 IEEE International Conference on Bioinformatics and Biomedicine (IEEE-BIBM 2017), Kansas City, MO, USA.</strong></li>                     
									<li> <strong><em> ad hoc </em>Reviewer:</strong></li>
									<ul class="major">
									        
											
									<div class="content span-7" style="margin-left:20px" style="margin-top:30px">
									    <ul class="major">
										<li><strong>  Frontiers in Oncology</strong> </li>
							                <li><strong>  IEEE Journal of Biomedical and Health Informatics</strong> </li>
										    <li> <strong> Frontiers in Genetics</strong></li>
											<li><strong>  Information Sciences</strong> </li>
											<li> <strong> PLoS One</strong></li>
											<li><strong>  Scientific Reports, Nature Publishing Group</strong> </li>
											<li> <strong> Applied Soft Computing</strong></li>
											<li> <strong> Engineering Applications of Artificial Intelligence</strong></li>
											<li> <strong> Computer Methods and Programs in Biomedicine</strong></li>
											<li> <strong> International Journal of Electrical Power & Energy Systems</strong></li>
											<li> <strong> Reliability Engineering and System Safety</strong></li>
											<li> <strong> Hydrological Sciences Journal</strong></li>
											<li> <strong> Journal of Intelligent and Fuzzy Systems</strong></li>
											<li> <strong> British Journal of Applied Science & Technology</strong></li>
											<li> <strong> International Journal of Engineering</strong></li>
											<li> <strong> Ain Shams Engineering Journal</strong></li>
											<li> <strong> North American Fuzzy Information Processing Society (IEEE-NAFIPS)</strong></li>
											<li> <strong> IEEE Microwave and Radio Electronics Week 2017 (IEEE MAREW 2017)</strong></li>										   
                                      </ul>
							      	</div>
											
									</ul>
									
									         
									<li> <strong> <em> Instructor</em>: 'Multiple Criteria Decision Making course', Graduate Course, Amirkabir University of Technology (International Campus), October 2012.</strong></li>		
                                    <li> <strong> <em> Instructor</em>: 'Foundations of Information Technology', Undergrad Course, Ershad University, Tehran, Iran.</strong></li>	
                                    <li> <strong> <em> Instructor</em>: 'Applied Optimization', Undergrad Course, Ershad University, Tehran, Iran, 2012.</strong></li>	
                                    <li> <strong> <em> Research Assistant</em>: Intelligent Systems Laboratory, Amirkabir University of Technology, 2011-2015.</strong></li>		
                                    <li> <strong> <em> Teaching Assistant</em>: 'Management of Information Systems' course of undergraduate level, Amirkabir University of Technology, 2013-2015.</strong></li>	
                                    <li> <strong> <em> Research Assistant</em>: 'Data Management', Amirkabir University of Technology, 2014-2015 .</strong></li>										
                                      </ul>
								</div>

							</section>
									
							
							<section id="ment" class="main ">
								<header class="major">
									<h2>Mentoring Experience</h2>
								</header>
                             <div class="content span-7" style="margin-left:20px" style="margin-top:30px">
									    <ul class="major">
							
										<li> <strong> <em> John Gregg</em>, PhD Student of Genomics and Computational Biology, University of Pennsylvania, 2018 (co-mentored).</strong></li>
										<li> <strong> <em> Karen Xu</em>, MS Student of Biostatistics, Columbia University, 2017 (co-mentored).</strong> </li>	
	                                   <li> <strong> <em> Eshan Agarwal</em>, BS Student of Computer Science, Columbia University, 2017.</strong></li>		
                                       <li> <strong> <em> Richard Jiang</em>, PhD Student of Computer Science, University of California, Santa Barbara (UCSB), 2016.</strong></li>	
                                     <li> <strong> <em> Destinee Cheng</em>, PhD Student of Computer Science, University of California, Santa Barbara (UCSB), 2016.</strong></li>										   
                                      </ul>
								</div>

							</section>
							
						
							
						<!-- First Section -->
							<section id="first" class="main special">
								<header class="major">
									<h2>Social Footprints</h2>
									<p>My professional accounts.  I use my Twitter primarily for science. My Github and Linkedin profiles project my other scientific achievements. Finally, I have recently linked my Instagram to my website.</p>
								</header>
								<ul class="feature">
								<span><a href="https://twitter.com/a_doostparast"  class="fa fa-twitter" style="font-size:120px; color:rgb(0, 132, 180); text-decoration:none;"></a></span> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
                                <span><a href="https://github.com/adoostparast" class="fa fa-github" style="font-size:120px; color:rgb(33, 31, 31)"></a></span> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
								<span><a href="https://www.linkedin.com/in/abolfazl-doostparast-68054534" class="fa fa-linkedin-square" style="font-size:120px; color:rgb(0, 119, 181)"></a></span>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
								<span><a href="https://www.researchgate.net/profile/Abolfazl_Doostparast" class="fab fa-researchgate" style="font-size:120px"></a></span>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; 
                                <span><a href="https://www.instagram.com/nasserdoostparast/" class="fa fa-instagram" style="font-size:120px;color:rgb(88, 81, 219)"></a></span>
							

								</ul>

							</section>

						



					</div>

				<!-- Footer -->
					<footer id="footer">


						<p class="copyright">&copy; Untitled. Design: <a href="https://html5up.net">HTML5 UP</a>.</p>
					</footer>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
